ecosystem/sep-0038.md

```diff
@@ -230,11 +230,11 @@ Optional [`SEP-10`]. If authentication is provided it can be used to personalize
 #### Request
 
 | Name                   | Type   | Description                                                                                                                                                                                                                                                                                                |
-| ---------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| ---------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Is this suppose to change?

Reply:
The table is broken. This is to fix it
![Screenshot 2024-09-25 at 3 31 34 PM](https://github.com/user-attachments/assets/3cf875ca-aac9-4171-ae26-9001bdbee518)

Reply:
Nice